verifiers/proofofsql/src/errors.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub struct LibraryError(proof_of_sql_verifier::VerifyError);
```

Comment:
Use `LibraryError` is fine for me... But you can also use `VerifyError` error here and then you have two options
1. Import it in `lib.rs` and rename it like `use errors::VerifyError as LibraryError`
2. Use `errors::VerifyError` instead of `LibraryError` in `lib.rs`